contracts/SlashIndicator.sol

```diff
@@ -197,6 +234,13 @@ contract SlashIndicator is ISlashIndicator,System,IParamSubscriber, IApplication
     emit indicatorCleaned();
   }
 
+  function downtimeSlash(address validator, uint256 count) public override {
+    try IStakeHub(STAKE_HUB_ADDR).downtimeSlash(validator) {}
+    catch (bytes memory reason) {
+      emit failedFelony(validator, count, reason);
+    }
+  }
+
   function submitFinalityViolationEvidence(FinalityEvidence memory _evidence) public onlyInit onlyRelayer {
```

Comment:
`submitFinalityViolationEvidence` this function need redesign, as part of the check is delegated onto BC, but after this hardfork, BSC should do it by itself. Please talk to Nathan.

Reply:
Done